plugins/PrivacyManager/SystemSettings.php

```diff
@@ -53,12 +61,35 @@ private function createTermsAndConditionUrlSetting()
         });
     }
 
-    private function createShowInEmbeddedWidgetsSetting()
+    private function createShowInEmbeddedWidgetsSetting(): SystemSetting
     {
         return $this->makeSetting('showInEmbeddedWidgets', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
             $field->title = Piwik::translate('PrivacyManager_ShowInEmbeddedWidgets');
             $field->uiControl = FieldConfig::UI_CONTROL_CHECKBOX;
             $field->description = Piwik::translate('PrivacyManager_ShowInEmbeddedWidgetsDescription');
         });
     }
+
+    private function createForceCookielessTrackingSetting(): SystemConfigSetting
+    {
+        return $this->makeSettingManagedInConfigOnly('Tracker', 'force_cookieless_tracking', $default = false, FieldConfig::TYPE_BOOL, function (FieldConfig $field) {
+            $field->title = Piwik::translate('PrivacyManager_ForceCookielessTracking');
+            $field->uiControl = FieldConfig::UI_CONTROL_CHECKBOX;
+            $field->description = Piwik::translate('PrivacyManager_ForceCookielessTrackingDescription');
```

Comment:
here would probably also need to check if matomo.js is writable

Reply:
What shall we do if it is not writable? Shall we add an additional note, that it won't work fully or should we even hide the setting?